### ISSUE TITLE
Create .deb packages for Ubuntu

### ISSUE THREAD

**Issue by TheJaredWilcurt** [scout-app/scout-app#257]
On Ubuntu, you can download `.deb` files, double click them, and they'll just magically install. Super convenient . No idea how to do that.

Example:

* http://sublimetext.com/3

**Feature Status**:

1. Will accept pull requests that improve our `build.js` script for automated Linux builds.
1. Will listen to community feedback and requests for this feature.
1. Very low priority. Unless I get feedback/requests for this, I will assume that Linux users are fine using the `.zip` method of install we currently use.

**Comment by adlibeo** [comment]
I'm migrating from MacOS to Elementary OS. I've used Scout on MacoS and it was a simple install, would love a similar (simple and straightforward install) on Linux

**Comment by Paras16699** [comment]
Any update on this?

**Comment by TheJaredWilcurt** [comment]
Still desired. Still low priority. Still waiting on community PR.

**Comment by fsrocha-dev** [comment]
How to install Scout-app in ubuntu ? please help

**Comment by TheJaredWilcurt** [comment]
1. Download the correct Zip file (32 or 64)
1. Unzip
1. Double-Click on `Scout-App` executable
1. If that doesn't work, you may need to open via command line with `./Scout-App`

**Comment by fsrocha-dev** [comment]
Thanks
